translate.js

```diff
@@ -38,6 +38,82 @@ const colorHelper = (colors, color, shade) => {
   );
 };
 
+// Adopted from `react-colorful` here: https://github.com/omgovich/react-colorful/blob/master/src/utils/convert.ts
```

Comment:
Alright.. I love how flexible this is; handling hex, rgb, rgba and keyword colors. But I'm wondering wether it is required. Would it be unreasonable to ask that devs convert their colors to hex before adding them to the theme?

Feels like this might add a lot on unnecessary weight and edge cases without offering that much value in terms of convenience. What are your thoughts as a consumer of the library?

Reply:
For my consumption, totally happy to stay within the realm of hex. That's all I use.

I thought I'd cover all the bases first, reduce later, because you're right. `rgb()` is just verbose hex and most of the keywords would be redundant (basic colors already accessible, would only be using the odd stuff like `olive`).

`hsl` I could maybe see some use for, but equally I don't think it's a big ask to convert into hex. A site won't have *that* many colors.

Reply:
Awesome, then yeh.. we are in agreement. Happy to support just hex until a valid use case comes along!